environments/icds-cas/postgresql.yml

```diff
@@ -1,7 +1,11 @@
 DEFAULT_POSTGRESQL_HOST: pgmain0
 REPORTING_DATABASES:
   ucr: ucr
-  icds-ucr-non-dashboard: icds-ucr-non-dashboard
+  icds-ucr-non-dashboard:
+    WRITE: icds-ucr-non-dashboard
+    READ:
+      - [icds-ucr-non-dashboard, 1]
+      - [icds-ndb-ucr-standby0, 10]
```

Comment:
perhaps best to leave this out for now until we know that the standby won't be impacted by promoting the master

Reply:
Sure, updated